evalml/automl/automl_algorithm/default_algorithm.py

```diff
@@ -326,8 +289,14 @@ def _create_n_pipelines(self, pipelines, n):
                     self._create_tuner(pipeline)
 
                 select_parameters = self._create_select_parameters()
-                proposed_parameters = self._tuners[pipeline.name].propose()
-                parameters = self._transform_parameters(pipeline, proposed_parameters)
+                parameters = (
```

Comment:
Fixed the parameters we use when we call this through `_create_fast_final` versus `_create_n_pipelines` with a larger size of pipelines